src/Auth0.php

```diff
@@ -382,6 +359,12 @@ public function __construct(array $config)
             $this->stateHandler = new SessionStateHandler($stateStore);
         }
 
+        if (isset($config['cache_handler']) && $config['cache_handler']) {
+            $this->cacheHandler = $config['cache_handler'];
+        } else {
+            $this->cacheHandler = new NoCacheHandler();
+        }
```

Comment:
```php
$this->cacheHandler = $config['cache_handler'] ?? new NoCacheHandler();
```

?

Reply:
Better, I agree (still getting used to the fact that I can use null coalescing now). I also need to check that it's the right instance, though. 

I'm going to deal with the cache handling in another PR but thanks for pointing this out 👍